kpi/tests/api/v2/test_api_assets.py

```diff
@@ -311,6 +299,44 @@ def test_asset_version_id_and_content_hash(self):
         self.assertEqual(response.data['version__content_hash'],
                          self.asset.latest_version.content_hash)
 
+    def test_assignable_permissions(self):
+        self.assertEqual(self.asset.asset_type, 'survey')
+        response = self.client.get(self.asset_url, format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertListEqual(
+            response.data['assignable_permissions'],
+            [
+                {'label': u'View form',
+                 'url': 'http://testserver/api/v2/permissions/view_asset/'},
+                {'label': u'Add submissions',
+                 'url': 'http://testserver/api/v2/permissions/add_submissions/'},
+                {'label': u'View submissions',
+                 'url': 'http://testserver/api/v2/permissions/view_submissions/'},
+                {'label': u'Change submissions',
```

Comment:
Does not pass the tests because the label is `Change and delete submissions` now.

Reply:
Oops